tests/query/v1/test_find_path.py

```diff
@@ -379,3 +379,158 @@ def test_multi_source_no_path(self):
         }
         self.check_column_names(resp, expected_data["column_names"])
         self.check_empty_result(resp)
+
+    def test_run_time_input(self):
+        stmt = '''YIELD "Yao Ming" AS src, "Tony Parker" AS dst
+                | FIND SHORTEST PATH FROM $-.src TO $-.dst OVER like, serve UPTO 5 STEPS'''
+        resp = self.execute_query(stmt)
+        self.check_resp_succeeded(resp)
+        expected_data = {
+            "column_names": ["_path"],
+            "rows": [
+                [b"Yao Ming", (b"like", 0, b"Shaquile O'Neal"), (b"like", 0, b"Tim Duncan"), (b"like", 0, b"Tony Parker")],
```

Comment:
Why not handle the binary string in check_path_xxx function for building the string's expected result easily.

Reply:
done